frontend/src/scenes/web-analytics/webAnalyticsLogic.tsx

```diff
@@ -927,6 +948,23 @@ export const webAnalyticsLogic = kea<webAnalyticsLogicType>([
                                           sessionsSeries,
                                       ])
                                     : null,
+                                !conversionGoal &&
+                                    revenueEventsSeries?.length &&
+                                    createGraphsTrendsTab(
+                                        GraphsTab.REVENUE_EVENTS,
+                                        'Revenue',
+                                        'Revenue',
+                                        revenueEventsSeries,
+                                        {
+                                            display:
+                                                revenueEventsSeries.length > 1
+                                                    ? ChartDisplayType.ActionsAreaGraph
+                                                    : ChartDisplayType.ActionsLineGraph,
+                                        },
+                                        {
+                                            compareFilter: revenueEventsSeries.length > 1 ? undefined : compareFilter,
+                                        }
```

Comment:
It'd be interesting to see comparison for multiple event series. Maybe the condition for Area vs Line should be `revenueEventsSeries.length > 1 && !compareFilter` instead, and then we only display area in case they're not comparing

Reply:
I tried it and found it was just way too noisy to be useful

Reply:
I can see that